app/helpers/application_helper.rb

```diff
@@ -881,4 +881,9 @@ def show_component(url) # rubocop:disable Metrics/CyclomaticComplexity TODO: Rem
       true
     end
   end
+
+  def display_my_broker?(person, employee_role)
+    employee_role ||= person.active_employee_roles.first
```

Comment:
Checks to see if employee_role exists and if not picks the first active employee role.